### PR TITLE
Calendar Reminders

### LINKED ISSUE #42: Add to "Schedule" Button
Instead of mark me interested, make a button drop down that lets people add the event to their calendar
The dropdown should have these calendar services for options: 
- Google Calendar 
- Other calendar (downloads .ics file for event)

The calendar event should have a link to the event as well

### PR DESCRIPTION
**Pull request by danielabreo**
closes #42

### CODE REVIEW

File: src/common/ActivityInfo/ActivityInfo.tsx [hunk 1]
```diff
@@ -79,6 +101,31 @@ export const ActivityInfo: React.FC<InfoProps> = (props: InfoProps) => {
...
+                  target="_blank"
+                  href={
+                    `https://www.google.com/calendar/render?` +
+                    `action=TEMPLATE&` +
+                    `text=${props.title}&` +
+                    `dates=${formatGoogleTime(
+                      props.startTime,
+                      props.endTime
+                    )}&` +
+                    `details=Attend Here: ${props.mediaLink}&` +
```
Comment (samarthdave) [inline src/common/ActivityInfo/ActivityInfo.tsx:119]: This renders the Zoom link directly which is the temporary link for now (```https://zoom.us/join/zoomRoomIdGoesHere```).
- Can we just link to the Portal event page? That way we can track the click eventually.
- Also, Can you do a conditional render and hide this button if the event has already passed?

File: src/common/ActivityInfo/ActivityInfo.tsx [hunk 2]
```diff
@@ -48,6 +73,64 @@ export const ActivityInfo: React.FC<InfoProps> = (props: InfoProps) => {
   //    */
   // };
 
+  const getICS = () => {
```
Comment (geooot) [inline src/common/ActivityInfo/ActivityInfo.tsx:76]: - Use React.useCallback
- Add comments to function to make it more obvious what it does.

File: src/common/ActivityInfo/ActivityInfo.tsx [hunk 3]
```diff
@@ -48,6 +73,64 @@ export const ActivityInfo: React.FC<InfoProps> = (props: InfoProps) => {
...
+          </Dropdown.Toggle>
+          <Dropdown.Menu style={{ width: "100%" }}>
+            <Dropdown.Item
+              target="_blank"
+              href={
+                `https://www.google.com/calendar/render?` +
+                `action=TEMPLATE&` +
+                `text=${props.title}&` +
+                `dates=${formatGoogleTime(props.startTime, props.endTime)}&` +
+                `details=Attend Here: tamudatathon.com/events&` +
```
Comment (geooot) [inline src/common/ActivityInfo/ActivityInfo.tsx:117]: Change url to directly go the the activity page.

File: src/common/ActivityInfo/ActivityInfo.tsx [hunk 4]
```diff
@@ -37,9 +47,24 @@ export const formatTime = (time: Date, endTime: Date): string => {
   );
 };
 
+export const formatGoogleTime = (startTime: Date, endTime: Date): string => {
+  /**
+   * Returns string in YYYYMMDDTHHmmSS/YYYYMMDDTHHmmSS format.
+   * @param startTime a date object
+   * @param endTime a date object
+   */
```
Comment (geooot) [inline src/common/ActivityInfo/ActivityInfo.tsx:55]: Put JS doc outside function

File: src/common/ActivityInfo/ActivityInfo.tsx [hunk 5]
```diff
@@ -48,6 +73,67 @@ export const ActivityInfo: React.FC<InfoProps> = (props: InfoProps) => {
...
+      }
+      const blob = new Blob([value], { type: "text/plain;charset=utf-8" });
+      FileSaver.saveAs(blob, "invite.ics");
+    });
+  }, []);
+
+  const remindMeMenu =
+    minsToEvent > 0 ? (
+      <>
+        <Dropdown>
```
Comment (geooot) [inline src/common/ActivityInfo/ActivityInfo.tsx:108]: Can we make the dropdown use an outline style?
image

Comment (danielabreo) [reply src/common/ActivityInfo/ActivityInfo.tsx:108]: Screen Shot 2020-08-12 at 3 23 32 PM

**Pull request merged**